paasta_tools/instance/kubernetes.py

```diff
@@ -325,7 +325,7 @@ async def mesh_status(
 
     pods = await pods_task
     for location, hosts in node_hostname_by_location.items():
-        host = replication_checker.get_first_host_in_pool(hosts, instance_pool)
+        host = replication_checker.get_hostname_in_pool(hosts, instance_pool)
```

Comment:
do we use `get_first_host_in_pool` for anything else that could cause issues in the future?

Reply:
https://sourcegraph.yelpcorp.com/search?q=get_first_host_in_pool&patternType=literal the only references are in paasta.

(though it looks like maybe there's a test that needs updating?)

Reply:
there's just tests for the get_* functions - i thought about adding one for this, but it seems like i'd really just be checking that random.choice was called?